src/Microsoft.AspNet.Mvc.TagHelpers/ImageTagHelper.cs

```diff
@@ -75,23 +75,20 @@ public class ImageTagHelper : UrlResolutionTagHelper
         /// <inheritdoc />
         public override void Process(TagHelperContext context, TagHelperOutput output)
         {
+            output.CopyHtmlAttribute(SrcAttributeName, context);
+            ProcessUrlAttribute(SrcAttributeName, output);
```

Comment:
:ok: I agree it can't be `null` due to the requirements in `[TargetElement]` and the way we generate code for required `string` attributes.